.env.example

```
@@ -24,6 +24,11 @@ POSTGRES_PASSWORD=shelfy
 # MinIO local bootstrap credentials
 MINIO_ROOT_USER=minioadmin
 MINIO_ROOT_PASSWORD=minioadmin
+MINIO_ENDPOINT=http://minio:9000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ENDPOINT key should go before the MINIO_ROOT_PASSWORD key

(UnorderedKey)